src/main/java/com/google/sps/servlets/MealInfoServlet.java

```diff
@@ -0,0 +1,47 @@
+// Copyright 2020 Google LLC
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     https://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
+import com.google.sps.data.Meal;
+
+@WebServlet("/meal-info")
+public class MealInfoServlet extends HttpServlet {
+
+  Meal meal;
+
+  // TODO(grenlayk): implement getting meal info by id (probably in another servlet according to REST API)
+
+  @Override
+  public void init() {
+    meal = new Meal(new Long(0), "Ice cream", "Chocolate ice cream with strawberries", new ArrayList<String>(Arrays.asList("Ice cream", "Strawberry")), "Dessert");
```

Comment:
Use `0L` instead of `new Long(0)`, it'll be cleaner.

As `Long` is compatible with `long` and not `int`, you need to use long literals in code. Without any extras, a set of digits is a literal `int`. Adding an extra capital `L` at the end makes it a literal `long` instead, which will match your `Meal(Long, ...)` constructor.